Makefile

```diff
@@ -91,6 +91,9 @@ pull_translations: ## pull translations from Transifex
 push_translations: ## push source translation files (.po) from Transifex
 	tx push -s
 
+test_translations: ## Test translation files
+	cd edx_proctoring && i18n_tool validate
```

Comment:
Why not include this in the existing `validate_translations` target?

Reply:
validate_translations uses other make targets too and using it will enforce other checks too. thats why i avoided it as its not being used for testing before.